src/utils/data/input.json

```diff
@@ -0,0 +1,168 @@
+{
+  "selected_pipeline": "__default__",
```

Comment:
I've just removed `input.json` in https://github.com/quantumblacklabs/kedro-viz/pull/281, and started using animal test data, so you can delete this file :)

Reply:
awesome, thanks Kiyo!